docker-compose.yaml

```diff
@@ -113,6 +113,8 @@ services:
     image: nvmeof-builder-base
     build:
       target: builder-base
+      args:
+        NVMEOF_TARGET: cli
```

Comment:
@epuertat how are you? Is this `docker-compose.yaml` change related to gRPC update?

Reply:
Yup, if you don't set this, then it looks for the `python3-librados`, and that's not strictly needed neither for the `cli` or the `gw` services, as that needs to be installed via `dnf`, not from PyPi.